src/timeline/Timeline.tsx

```diff
@@ -247,7 +252,7 @@ const Timeline = (props: TimelineProps) => {
       // @ts-expect-error
       ref={scrollView}
       style={styles.current.container}
-      contentContainerStyle={[styles.current.contentStyle, {width: constants.screenWidth}]}
+      contentContainerStyle={[styles.current.contentStyle, {width: (availableWidth ?? constants.screenWidth)}]}
```

Comment:
I would prefer not to have a prop just for the content container width but instead, allow overriding/merge to the 'contentContainerStyle'. Maybe by exposing 'scrollViewProps' prop where you can pass any ScrollView props and override some (!) of its props.